lightning/src/ln/channel.rs

```diff
@@ -1901,6 +1901,16 @@ impl<Signer: Sign> Channel<Signer> {
 		Ok(())
 	}
 
+	/// Returns transaction if there is pending funding transaction that is yet to broadcast
+	pub fn has_funding_pending(&self) -> Option<Transaction> {
```

Comment:
suggestion: prefer rename to just `pending_funding` (since it doesn't return a bool), or preferably `unbroadcasted_funding`